Tests/Contoso.Forms.Test.Strings/Strings.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
```

Comment:
Just curious, does xamarin forms has a proper way to handle localizable strings?